CPT331.Web/Controllers/AdminController.cs

```diff
@@ -9,44 +9,44 @@
 using CPT331.Core.ObjectModel;
 using CPT331.Data;
 using CPT331.Web.Models.Admin;
+using CPT331.Web.Actions;
 using CPT331.Web.Attributes;
 
 #endregion
 
 namespace CPT331.Web.Controllers
 {
-    /// <summary>
-    ///    A controller that provides methods that respond to HTTP requests that are related 
-    ///    to the administration portal.
-    /// </summary>
-    /// <permission cref="AdminAuthorizeAttribute">Only authorised personnel have access to methods in this controller.</permission>
-    [AdminAuthorize]
-    public class AdminController : Controller
-    {
-
-        /// <summary>
-        /// Provides a landing page for the administration portal.
-        /// </summary>
-        /// <returns>The 'Admin/Home' view.</returns>
-        /// <permission cref="AdminAuthorizeAttribute">Only authorised personnel have access to this method.</permission>
-        [HttpGet]
-        public ActionResult Home()
-        {
-            return View();
-        }
-
-        #region Crime Model
-        /// <summary>
-        /// Provides an editor page for an existing Crime record.
-        /// </summary>
-        /// <param name="id">The unique ID for the Crime record.</param>
-        /// <returns>The 'Edit Crime' view.</returns>
-        /// <permission cref="AdminAuthorizeAttribute">Only authorised personnel have access to this method.</permission>
-        [HttpGet]
+	/// <summary>
```

Comment:
One messy merge due to the reordering of methods and such. It should be all in there though.